doc/nrf/releases_and_maturity/releases/release-notes-3.1.0.rst

```diff
@@ -0,0 +1,1320 @@
+  * :ref:`nfc_shell`
+
+* :ref:`record_text` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l15/cpuapp/ns`` board target.
+
+nRF5340 samples
+---------------
+
+|no_changes_yet_note|
+
+Peripheral samples
+------------------
+
+* :ref:`radio_test` sample:
+
+  * Added experimental ``llvm`` toolchain support for the ``nrf54l15dk/nrf54l15/cpuapp`` board target.
+
+* :ref:`802154_phy_test` sample:
+
+  * Added print of sent packets and received Acks after ``ltx`` command.
+
+PMIC samples
+------------
+
+* :ref:`npm1300_fuel_gauge` sample:
+
+  * Updated current sign when calling the ``nrf_fuel_gauge`` lib functions.
+    In the Zephyr sensor API the convention for the gauge current is ``negative=discharging``, while the :ref:`nrfxlib:nrf_fuel_gauge` lib uses the opposite.
+
+Protocol serialization samples
+------------------------------
+
+|no_changes_yet_note|
+
+SDFW samples
+------------
+
+|no_changes_yet_note|
+
+Sensor samples
+--------------
+
+|no_changes_yet_note|
+
+SUIT samples
+------------
+
+* Removed all SUIT samples:
+
+  * SUIT: Device firmware “A/B” update on the nRF54H20 SoC
+  * SUIT: Flash companion
+  * SUIT: Recovery application
+  * SUIT: Device firmware update on the nRF54H20 SoC
+
+Trusted Firmware-M (TF-M) samples
+---------------------------------
+
+* :ref:`tfm_secure_peripheral_partition` sample:
+
+  * Added support for the ``nrf54l15dk/nrf54l15/cpuapp/ns`` board target.
+
+Thread samples
+--------------
+
+* Added the new :ref:`architecture option <openthread_stack_architecture>` to use the OpenThread stack directly to communicate with the IEEE 802.15.4 radio driver in the following samples:
+
+  * :ref:`ot_coprocessor_sample`
+  * :ref:`coap_server_sample`
+  * :ref:`ot_cli_sample`
+
+Wi-Fi samples
+-------------
+
+* Added runtime support for Buletooth LE based Wi-Fi provisioning in enterprise mode.
+* Removed system mode and the ``wpa_supplicant`` dependency for monitor mode optimization.
+
+* :ref:`wifi_radiotest_samples`:
+
+  * Updated :ref:`wifi_radio_test` and :ref:`wifi_radio_test_sd` samples to clarify platform support for single-domain and multi-domain radio tests.
+
+* :ref:`wifi_shutdown_sample`:
+
+  * Updated the sample to include both One-shot and Continuous modes of operations.
+
+Other samples
+-------------
+
+* Added the :ref:`mcuboot_minimal_configuration` sample that demonstrates the minimal and recommended settings for MCUboot on the nRF54L15 DK.
+
+Drivers
+=======
+
+This section provides detailed lists of changes by :ref:`driver <drivers>`.
+
+* Added the :ref:`mspi_sqspi` that allows for communication with devices that use MSPI bus-based Zephyr drivers.
+
+Wi-Fi drivers
+-------------
+
+|no_changes_yet_note|
+
+Flash drivers
+-------------
+
+* Removed the SUIT flash IPUC driver.
+
+Libraries
+=========
+
+This section provides detailed lists of changes by :ref:`library <libraries>`.
+
+Binary libraries
+----------------
+
+|no_changes_yet_note|
+
+Bluetooth libraries and services
```

Comment:
Please add for https://github.com/nrfconnect/sdk-nrf/pull/23552 PR, (@krish2718 to confirm):

* Updated the Wi-Fi Provisioning Service library by splitting it into the :ref:`lib_wifi_prov_ble` library and :ref:`lib_wifi_prov_core` library .
    The :ref:`lib_wifi_prov_core` library is now a standalone library that can be used by applications to implement the Wi-Fi provisioning functionality independently of the Bluetooth stack.